src/app/pages/registration/registration-form/registration-form.component.html

```diff
@@ -5,99 +5,74 @@ <h2>Registration Form</h2>
+          <mat-option value="S">S</mat-option>
+          <mat-option value="M">M</mat-option>
+          <mat-option value="L">L</mat-option>
+          <mat-option value="XL">XL</mat-option>
+          <mat-option value="XXL">XXL</mat-option>
+        </mat-select>
+        <mat-error *ngIf="size.invalid && (size.dirty || size.touched || submitted)">
           This field is required
-        </div>
-      </div>
-      <div class="form-group required">
-        <label>sex</label>
-        <label class="checkbox">
-          <input type="radio" name="sex" value="F" required [(ngModel)]="model.gender" #sex="ngModel"> female
-        </label>
-        <label class="checkbox">
-          <input type="radio" name="sex" value="M" required [(ngModel)]="model.gender" #sex="ngModel"> male
-        </label>
-        <div class="alert alert-danger" role="alert" *ngIf="sex.errors?.required && (sex.touched || submitted)">
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field>
+        <mat-select name="experience" id="experience" formControlName="experience" required placeholder="your experience">
+          <mat-option></mat-option>
+          <mat-option value="I am just learning">I am just learning</mat-option>
+          <mat-option value="Less than a year">Less than a year</mat-option>
+          <mat-option value="1-2 years">1-2 years</mat-option>
+          <mat-option value="2-4 years">2-4 years</mat-option>
+          <mat-option value="4-8 years">4-8 years</mat-option>
+          <mat-option value="Over 8 years">Over 8 years</mat-option>
+        </mat-select>
+        <mat-error *ngIf="experience.invalid && (experience.dirty || experience.touched || submitted)">
           This field is required
-        </div>
-      </div>
-      <div class="form-group required">
-        <label for="size">t-shirt size</label>
-        <select name="size" id="size" class="form-control" [(ngModel)]="model.size" #size="ngModel" required>
-          <option></option>
-          <option value="S">S</option>
-          <option value="M">M</option>
-          <option value="L">L</option>
-          <option value="XL">XL</option>
-          <option value="XXL">XXL</option>
-        </select>
-        <div class="alert alert-danger" role="alert" *ngIf="size.errors?.required && (size.touched || submitted)">
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field>
+        <mat-select name="role" id="role" formControlName="role" required placeholder="your role" >
+          <mat-option></mat-option>
+          <mat-option value="student">Student</mat-option>
+          <mat-option value="Developer">Developer</mat-option>
+          <mat-option value="Team Leader">Team Leader</mat-option>
+          <mat-option value="Project Leader">Project Leader</mat-option>
+          <mat-option value="Architect">Architect</mat-option>
+          <mat-option value="Tester">Tester</mat-option>
+          <mat-option value="Other">Other</mat-option>
```

Comment:
I am wondering if we can ask this data now (GDPR). We dont need this data to do our business, so for me it is questionable.